VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -0,0 +1,143 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
+
+```
+ric@getmonero.org
+BDA6 BD70 42B7 21C4 67A9  759D 7455 C5E3 C0CD CEB9
+
+luigi1111@getmonero.org
+8777 AB8F 778E E894 87A2  F8E7 F4AC A018 3641 E010
+
+moneromooo.monero@gmail.com
+48B0 8161 FBDA DFE3 93AD  FC3E 686F 0745 4D6C EFC3
+```
+
+## II. Security Response Team
+
+- fluffypony
+- luigi1111
+- moneromooo
+
+## III. Incident Response
+
+1. Researcher submits report via one or both of two methods:
+    - a. Email
+    - b. [HackerOne](https://hackerone.com/monero)
+
+2. Response Team designates a Response Manager who is in charge of the particular report based on availability and/or knowledge-set
+
+3. In no more than 3 working days, Response Team should gratefully respond to researcher using only encrypted, secure channels
+
+4. Response Manager makes inquiries to satisfy any needed information to confirm if submission is indeed a vulnerability
+    - a. If submission proves to be vulnerable, proceed to next step
+    - b. If not vulnerable:
+      - i. Response Manager responds with reasons why submission is not a vulnerability
+      - ii. Response Manager moves discussion to a new or existing ticket on GitHub if necessary
+
+5. If over email, Response Manager opens a HackerOne issue for new submission
+
+6. Establish severity of vulnerability:
+    - a. HIGH: effects network as a whole, has potential to break entire network, results in the loss of monero, or is on a scale of great catastrophe
+    - b. MEDIUM: effects individual nodes, wallets, or must be carefully exploited
```

Comment:
Why it's *effect*: the colon is a substitute for "that" or "which has an effect", which in turn treats the listed points as consequences to the antecedent. Establishing severity of vulnerability does not "act upon; to produce an effect or change upon" the resulting consequence but rather is a response *to* the affect of exploitation:

> Syn: {Effect}, {Consequence}, {Result}.
>  Usage: These words indicate things which arise out of some
          antecedent, or follow as a consequent. Effect, which
          may be regarded as the generic term, denotes that
          which springs directly from something which can
          properly be termed a cause. A consequence is more
          remote, not being strictly caused, nor yet a mere
          sequence, but following out of and following
          indirectly, or in the train of events, something on
          which it truly depends.

>  {Of no effect}, {Of none effect}, {To no effect}, or {Without
   effect}, destitute of results, validity, force, and the like;
      vain; fruitless. `Making the word of God of none effect
      through your tradition.` --Mark vii. 13. `All my study
      be to no effect.` --Shakespeare

> {To give effect to}, to make valid; to carry out in practice;
      to push to its results.

>{To take effect}, to become operative, to accomplish aims.
      --Shakespeare

So either we drop the colon and clarify, or trust the language 😄